gears/src/rest/tendermint_events_handler.rs

```diff
@@ -0,0 +1,33 @@
+// TODO: update errors
+use anyhow::anyhow;
+
+pub struct StrEventsHandler<'a> {
+    events_str: &'a str,
+}
+
+impl<'a> StrEventsHandler<'a> {
+    pub fn new(events_str: &'a str) -> Self {
+        let events_str = events_str.trim_start_matches('\'').trim_end_matches('\'');
+        Self { events_str }
+    }
+
+    pub fn try_parse_tendermint_events_vec(&self) -> anyhow::Result<Vec<String>> {
```

Comment:
Would it make sense to return `Vec<(String : attribute, String : value)>` instead of string concatenation?

Reply:
We could use this solution but it's much easier to use Query parser.